lib/mhv/api/new_users.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module MHV
+  module API
+    module NewUsers
+      def post_user_registration(attributes)
+        user = MHV::User.new(attributes)
```

Comment:
For EVSS, we're just using a User model in /app/models. Should we be putting ours in a namespace somewhere? My thought was that we were aiming for a unified user model, but maybe it makes sense to isolate the properties.